WebRoot/css/common.css

```diff
@@ -589,6 +589,7 @@ LI.RowDouble, LI.Row {list-style: none;}
 /* .Row-matched TD			 	{ 	@ListItemText-matched@				} */
 
 .Row-focused TD 				{	@ListItemText-focused@			}
+.Row-selected.Row-focused TD 	{	@ListItemText-selected@			}
```

Comment:
I can add more tabs to match the above line. Though its not very good with tabs in here :)